src/fuse_api.pxi

```diff
@@ -467,6 +467,25 @@ cdef session_loop_mt(workers):
         stdlib.free(wd)
 
 
+def stop_fuse_from_another_thread():
+    '''Stop the FUSE main loop from another thread.
+    '''
+    res = pthread_mutex_lock(&exc_info_mutex)
```

Comment:
You're not doing anything with exc_info (which is what this mutex protects), so no need to acquire it.

Reply:
done.

Reply:
so be it ^^